docs/_future/deploy_your_app.md

```diff
@@ -7,10 +7,212 @@ layout: tutorial
+$ heroku config:set SLACK_BOT_TOKEN=xoxb-2b07-4pp5hv3f3ll85
+```
+
+These tokens and variables don't necessarily have to match those used in development. For instance, you may want to use tokens for a different app or a more verbose logging output in production. This is where you would set that.
+
+### Prepare your app for Heroku {#prepare-your-app}
+
+No changes to your app should be needed to deploy, but a few configurations must be made to make a deployment successful.
+
+#### Add a Procfile
+
+To specify the start command of your app for Heroku, [a special file called **Procfile**](https://devcenter.heroku.com/articles/procfile) must be created.
+
+The contents of this file will vary depending if you are using Socket Mode or not, but in both cases it will be named `Procfile` and live in your app's root directory.
+
+If you are using Socket Mode, your `Procfile` should contain the following:
+```sh
+worker: node app.js
+```
+
+A Slack app using default web connections will have the following `Procfile`:
+
+```sh
+web: node app.js
+```
+
+With this new file saved, go ahead and commit it to your Git repository:
+
+```sh
+$ git add Procfile
+$ git commit -m "Add Procfile"
+```
+
+#### Add the Git remote
+
+Code is deployed to Heroku by pushing code from your local repository to the Git remote from when [you created an app](#create-a-heroku-app).
+
+You can check if this remote was automatically added by the Heroku CLI when your app was created like so:
+
+```sh
+$ git remote -v
+# heroku  https://git.heroku.com/aqueous-escarpment-85734.git (fetch)
+# heroku  https://git.heroku.com/aqueous-escarpment-85734.git (push)
+```
+
+But if there is no `heroku` remote, you can add one as follows:
+
+```sh
+$ git remote add heroku https://git.heroku.com/aqueous-escarpment-85734.git
+```
+
+### Deploy your app {#deploy-your-app}
+
+After creating and configuring your Bolt and Heroku app, you're ready for deployment!
+
+> 💾 Any [triggers created with the CLI](https://api.slack.com/future/triggers#create_cli) for the local "(dev)" version of your app will continue to fire after deploying your app!
+
+#### Push your code
+
+But before you can deploy, you should ensure that you have committed all of the source code your app needs to run!
+
+You can see unstaged changes using `git status` and commit any changes using the same commands used for **Procfile**.
+
+After adding and committing the source code of your app to the `main` branch of your repo, push it to your Heroku remote to start a deployment:
+
+```sh
+$ git push -u heroku main
+```
+
+#### Start your app
+
+To kick things off on your Heroku app, you might have to scale up the dyno that runs your app.
+
+For socket mode, this can be done by stopping the `web` dyno and starting the `worker` dyno:
+
+```sh
+$ heroku ps:scale web=0 worker=1
+```
+
+While apps using web connections can be started with just the `web` dyno:
+
+```sh
+$ heroku ps:scale web=1
+```
+
+#### Using a public web address
```

Comment:
Do you think leaving this out of the header would be alright since the "web connections" note is shared below? The extra line length is causing the header to break lines on smaller screens which isn't really a problem, I just have a preference for the one-line header if possible!

Reply:
after reading through this again - what are your thoughts on updating this header title to make it more explicit that this is only a required step if your app isn't using socket mode? I'm not sure what that would look like, maybe something like "Settings for apps not using Socket Mode" but with a bit more indication about what's inside this section?